app/main.dev.ts

```diff
@@ -144,9 +150,19 @@ app.on( 'ready', async () => {
         }
     }
 
-    await setupBackground();
+    if ( !process.argv.includes( '--trigger-update' ) ) {
+        await setupBackground();
 
-    mainWindow = openWindow( store );
+        mainWindow = openWindow( store );
+    }
+
+    if ( await checkIfAppIsRunning() ) {
```

Comment:
`checkIfAppIsRunning` seems badly named. What do we want / expect to be returned here? Which app? the browser check is done via the `getInstanceLock`...
 
Maybe `XIsRunning` is a better name? (maybe not)

Reply:
No this checking if SNAPP is running and only then send a response to SNAPP. Cause otherwise it would always open SNAPP

Reply:
So let's name it appropriately (also considering the context it's to be used in as a bool)